.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 package-lock.json
 yarn.lock
+App.js
```

Comment:
@AleksandarVucenic whis is `App.js` ignored?

Reply:
App.js is ignored because it generates from Webpack

Reply:
If yould be great to ignore whole folder that contains `App.js` (if I am correct `dist` folder?).

Reply:
It's in the root; this `.gitignore` change is fine.